jit/v2/JIT_v2.0.0.yaml

```diff
@@ -826,29 +1004,27 @@ paths:
               examples:
                 badRequestExample:
                   summary: |
-                    Declining a Port Call Service
+                    Timestamp missing classifierCode
                   description: |
-                    `isFYI` is a boolean property - cannot set the value to '12'. This is an example of how the error object would look in case the value of this property is specified wrongly
+                    `classifierCode` is a mandatory property in the `Timestamp`. This is an example of how the error object would look in case this property is missing
 
-                    **NB**: `errorCode` not yet standardized by DCSA. Value `7004` is just a "random example"
+                    **NB**: `errorCode` not yet standardized by DCSA. Value `7003` is just a "random example"
                   value:
-                    httpMethod: POST
-                    requestUri: /v2/port-call-services/085a3207-5e45-49cf-8e1b-f8442beaf545/decline
+                    httpMethod: PUT
+                    requestUri: /v2/timestamps/085a3207-5e45-49cf-8e1b-f8442beaf545
                     statusCode: 400
                     statusCodeText: Bad Request
-                    statusCodeMessage: |
-                      isFYI has wrong type - boolean expected but integer found
+                    statusCodeMessage: classifierCode property not found - it is a mandatory property in Timestamp
                     providerCorrelationReference: 4426d965-0dd8-4005-8c63-dc68b01c4962
                     errorDateTime: '2024-11-21T09:41:00Z'
                     errors:
-                      - errorCode: 7004
-                        property: isFYI
-                        errorCodeText: wrong property type
-                        errorCodeMessage: |
-                          isFYI should be a boolean. '12' provided as value
+                      - errorCode: 7003
+                        property: classifierCode
+                        errorCodeText: mandatory property missing
+                        errorCodeMessage: classifierCode must be provided as part of a Timestamp
         '404':
           description: |
-            In case the provider does not know of the `portCallServiceID` used in the request (this could be because of a `PUT` request that has not finished processing or simply because the resource does not exist) - it is possible for the provider to reject the requests by returning a `404` (Not Found).
+            In case the provider does not know of the `portCallServiceID` used in the request, it is possible for the provider to reject the requests by returning a `404` (Not Found). This could be because of a `PUT` request, for creating/initiating the Port Call Service, that has not finished processing or simply because the resource does not exist.
```

Comment:
timestampID

Reply:
This was on purpose as it is referring to the portCallServiceID inside the payload ;)